index.html

```diff
@@ -13,13 +13,2689 @@
+              </div>
+            </div>
+          </div>
+          <!-- /culture and travel -->
+        </div>
+      </div>
+
+      <!--  free sign up form on small screens-->
+      <div class="d-none d-sm-block d-md-none mt-3 mx-4 p-4 purple-background">
+        <div
+          class="d-flex flex-row align-items-baseline justify-content-center"
+        >
+          <h5 class="float-left mr-2 mt-2">
+            <span class="px-1 font-weight-bold text-light red-background"
+              >N</span
+            >
+          </h5>
+          <h5 class="font-weight-bold">
+            START YOUR DAY WITH OUR TOP 5 ARTICLES
+          </h5>
+        </div>
+        <div class="form-group">
+          <input type="text" class="form-control" placeholder="Email address" />
+        </div>
+        <button
+          class="my-2 btn btn-block red-background text-light"
+          type="button"
+        >
+          FREE SIGN UP >
+        </button>
+      </div>
+
+      <!-- in the magazine-->
+      <div class="container-fluid">
+        <div
+          class="border-bottom d-flex flex-row align-items-baseline my-4 justify-content-center"
+        >
+          <h5 class="mr-2 mt-2">
+            <span class="px-1 font-weight-bold text-light red-background"
+              >N</span
+            >
+          </h5>
+          <h5 class="font-weight-bold">
+            IN THE MAGAZINE
+          </h5>
+        </div>
+
+        <div class="row align-items-baseline">
+          <!-- magazine 1-->
+          <div class="pb-2 col-lg-3 col-md-3">
+            <div class="card">
+              <a href="#">
+                <img
+                  class="card-img-top position-relative img-fluid"
+                  src="https://d.newsweek.com/en/full/1605463/us-cover-july-10-17.webp?w=290&h=387&q=90&f=9e75a0a61adba5b4c47bd32b6991f48b"
+                  alt="picture"
+                />
+              </a>
+              <div class="card-img-overlay p-0 d-flex align-items-end">
+                <p
+                  class="position-absolute px-3 py-2 bg-light font-weight-bold text-red"
+                >
+                  JULY 17 ISSUE
+                </p>
+              </div>
+            </div>
+
+            <div class="card-body pt-2 p-0">
+              <h5 class="card-title d-flex justify-content-center">
+                <a href="#" class="font-weight-bold text-body"
+                  >SEE ALL FEATURES ></a
+                >
+              </h5>
+            </div>
+          </div>
+          <!-- /magazine 1-->
+          <!-- magazine 2-->
+          <div class="my-1 col-lg-3 col-md-3">
+            <div class="card my-1">
+              <a href="#">
+                <img
+                  class="card-img-top position-relative img-fluid"
+                  src="https://d.newsweek.com/en/full/1604216/good-list-banner.webp?w=466&h=311&q=90&f=2062901c3efd89779019feae0efa97da"
+                  alt="picture"
+                />
```

Comment:
#### [OPTIONAL]

Consider increasing the height of the images in `in the magazine` section.

| Source Page	| Clone Page |
|:---:|:---:|
|![Annotation 2020-07-11 160039](https://user-images.githubusercontent.com/57726348/87222418-1ba9ab80-c391-11ea-9048-6ee35cf3020d.jpg)|![Annotation 2020-07-11 155926](https://user-images.githubusercontent.com/57726348/87222421-25331380-c391-11ea-8ee9-f1673bf5b100.jpg)|